docs/README.md

```diff
@@ -4,4 +4,4 @@ To work on the docs:
 
 1. Install Jekyll (e.g. `gem install jekyll`)
 2. Run `sbt docs/makeMicrosite` from the root of this repo.
-3. `cd` to `.target/site` and run `jekyll serve`. Visit the docs site at `localhost:4000` in your browser.
\ No newline at end of file
+3. `cd` to `.target/site` and run `jekyll serve`. Visit the docs site at `localhost:4000/evilplot/` in your browser.
```

Comment:
On my machine, I had to go to `docs/target/site`.  (Since you said run `sbt docs/makeMicrosite` from *the root*, I assumed the `cd` was also from the root.

Reply:
ah, yes, it used to be from the root, but we unified the projects at some point and never updated this readme. good catch!

Reply:
Changing to docs/target/site